code-challenges.test.js

```diff
@@ -14,54 +14,181 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// const fibLength1 = 6
+// // Expected output: [1, 1, 2, 3, 5, 8]
+
+// const fibLength2 = 10
+// // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+
+// describe ("fibSequence"), () => {
+//       //it method describes what function does
+//       it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+//           const fibLength1 = 6
+//           //Expected output: [1, 1, 2, 3, 5, 8]
+//           const fibLength2 = 10
+//           //Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+          
+          
+//       //expect calls on function and compares result to determined expected output using .toEqual
+//       expect(fibSequence(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+//       expect(fibSequence(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+//       }
+//       )}
+      
+     
+//       console.log(fibSequence(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+
+     // ReferenceError: describe is not defined
+
+
+
 
 // b) Create the function that makes the test pass.
 
+//PsudoCode:
+// create a function (fibSequence)
+// that takes in a input ()
+// if input is greater than 2 (if i >2)
+// use a for loop?
+
+// add numbers together and push to new 
+// repeat (index) times
+//number must be greater than 2 though
+
+// const fibSequence = () => {
+//   const seqArray = [];
+
+//   let num1 = 1;
+//   let num2 = 1;
+
+//   for (let i = 0; i < num; i++) {
```

Comment:
You set up the for loop, however you are not utilizing it at any point.  You are heading in the right direction though and super close to having this!